doc/releasenotes.html

```diff
@@ -59,6 +59,55 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
 
 -->
 
+<h1 id="2023-12-17-r1">SAM 2023.12.17 r1, SSC 28x: March xx, 2024</h1>
+
+<p>This is the first maintenance update to <a href="#2023-12-17">SAM 2023.12.17</a>.</p>
+
+<ul>
+<li>Solar Resource
+<ul>
+<li>Support NSRDB weather file downloads from the Solar Resource page for locations outside the United States. (<a href="https://github.com/NREL/SAM/pull/1720">SAM 1720</a>, <a href="https://github.com/NREL/ssc/pull/1140">SSC 1140</a>)</br>&nbsp;&nbsp;<a href="https://github.com/NREL/SAM/issues/1534">Additional work is planned</a> for the Advanced NSRDB Download window for the second update to SAM 2023.12.17.</li>
+<li>Fix problem using latitude and longitude to download weather files from the Solar Resource page. (<a href="https://github.com/NREL/SAM/pull/1692">SAM 1692</a>)</li>
+</ul>
+</li>
+<li>Detailed PV and PVWatts
+<ul>
+<li>Fix PV land area calculations. (<a href="https://github.com/NREL/SAM/pull/1721">SAM 1721</a>, <a href="https://github.com/NREL/SAM/pull/1697">SAM 1697</a>)</li>
+<li>Fix PV time series shading data import from 3D shade calculator and overlapping inputs in Edit Shading window for PVWatts. (<a href="https://github.com/NREL/SAM/pull/1712">SAM 1712</a>)</li>
+<li>Fix Detailed PV non-linear self shading for vertical arrays to avoid negative POA irradiance values. (<a href="https://github.com/NREL/ssc/pull/1127">SSC 1127</a>)</li>
+<li>Fix issue with Detailed PV subhourly clipping loss when weather file has no DNI data. (<a href="https://github.com/NREL/ssc/pull/1126">SSC 1126</a>)</li>
+<li>Fix reporting of ground-reflected rear irradiance gains in loss diagram for Detailed PV model with bifacial modules. (<a href="https://github.com/NREL/ssc/pull/1125">SSC 1125</a>)</li>
+<li>Add snow slide coefficient to outputs for Detailed PV with snow model enabled. (<a href="https://github.com/NREL/ssc/pull/1128">SSC 1128</a>)</li>
+<li>Clean up PV subhourly clipping loss correction in user interface. (<a href="https://github.com/NREL/SAM/pull/1689">SAM 1689</a>)</li>
+<li>Fix diagrams on Detailed PV Shading and Layout page to show both portrait and landscape instead of duplicating portrait. (<a href="https://github.com/NREL/SAM/pull/1696">SAM 1696</a>)</li>
+</ul>
+</li>
+<li>Concentrating Solar Power (CSP)
```

Comment:
Additional issue not tagged with patch that was merged: https://github.com/NREL/SAM/pull/1711